packages/docs/docs/docs/guide_create2_2.4.md

```diff
@@ -0,0 +1,213 @@
+---
+id: create2
+title: Upgradable Create2 Contracts with zOS
+---
+
+
+
+Create2 opcode gives us the ability to calculate smart contracts addresses without actually deploying them on Ethereum blockchain. This opens up a lot of possibilities to improve [user onboarding and scalability](https://blog.zeppelinos.org/getting-the-most-out-of-create2/).
```

Comment:
```suggestion
The CREATE2 opcode gives us the ability to calculate smart contracts addresses without actually deploying them on the Ethereum blockchain. This opens up a lot of possibilities to improve [user onboarding and scalability](https://blog.zeppelinos.org/getting-the-most-out-of-create2/).
```

We'll have a grammar editor go through the text later, but please keep an eye out for these kind of errors!